pkg/querier/querier.go

```diff
@@ -90,3 +90,17 @@ type NetworkPolicyQueryFilter struct {
 	// The type of the original NetworkPolicy that the internal NetworkPolicy is created for.(K8sNP, CNP, ANP)
 	SourceType cpv1beta.NetworkPolicyType
 }
+
+type ServiceExternalIPStatusQuerier interface {
+	GetServiceExternalIPStatus() []ServiceExternalIPInfo
+}
+
+// ServiceExternalIPInfo is used to check the status of external IPs for Services.
```

Comment:
This sounds like the comment of `ServiceExternalIPStatusQuerier`

Reply:
updated.